src/elements/content-sidebar/DetailsSidebar.js

```diff
@@ -39,7 +39,9 @@ import type { ClassificationInfo, ContentInsights, FileAccessStats } from './flo
 import type { WithLoggerProps } from '../../common/types/logging';
 import type { ElementsErrorCallback, ErrorContextProps, ElementsXhrError } from '../../common/types/api';
 import type { BoxItem } from '../../common/types/core';
+import type { FeatureConfig } from '../common/feature-checking';
 import './DetailsSidebar.scss';
+import { isFeatureEnabled, withFeatureConsumer } from '../common/feature-checking';
```

Comment:
Let's move this line above the SCSS and type imports.

Reply:
Fixed